src/Component/Symbolizer/TextEditor/TextEditor.tsx

```diff
@@ -276,6 +280,7 @@ export class TextEditor extends React.Component<TextEditorProps> {
                   onChange: this.onLabelChange,
                   propName: 'value',
                   propValue: symbolizer.label || '',
+                  defaultValue: defaultValues?.TextEditor?.defaultTemplate,
```

Comment:
Minor: shouldn't that rather be `defaultLabel` to be consistent with the geostyler-style property?

Reply:
Fixed in bbf91de